### PR TITLE
Bruce the keyboard

### PR DESCRIPTION
**Pull request by josh-l-wang**
adding bruce the keyboard

### CODE REVIEW

**Comment by lesshonor** [comment]
[Previously...](https://github.com/vial-kb/vial-qmk/pull/595#pullrequestreview-1722085946)

My suggestion to you is to quit using [your copy of the `vial` branch](https://github.com/josh-l-wang/vial-qmk/tree/vial). Rename or delete it if you have to. Assuming your `upstream` is `vial-kb/vial-qmk`, when you start adding a new keyboard, run:
- `git fetch upstream`
- `git checkout -b new_branch_name upstream/vial`

File: keyboards/jlw/bruce_the_keyboard/config.h
Comment (lesshonor) [inline keyboards/jlw/bruce_the_keyboard/config.h:1]: You can delete this file, since it's not doing anything.

File: keyboards/jlw/vault35_katana/rgb.c
Comment (lesshonor) [inline keyboards/jlw/vault35_katana/rgb.c:1]: Please delete this file.

File: keyboards/jlw/bruce_the_keyboard/keymaps/vial/config.h [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+#pragma once
+
+#define VIAL_KEYBOARD_UID {0x5D, 0x03, 0x13, 0x17, 0xBA, 0x20, 0xF5, 0x16}
+#define VIAL_UNLOCK_COMBO_ROWS { 0, 2 }
+#define VIAL_UNLOCK_COMBO_COLS { 0, 9 }
+
+#define VIAL_COMBO_ENTRIES 69
+#define DYNAMIC_KEYMAP_LAYER_COUNT 8
+#define TAPPING_TERM 200
+
```
Comment (lesshonor) [inline keyboards/jlw/bruce_the_keyboard/keymaps/vial/config.h:12]: Not a huge deal, but technically this isn't necessary as it is default.
```suggestion
```

**Review by lesshonor — CHANGES_REQUESTED** [review]

**Comment by xyzz** [comment]
Closing because there's been no update for over a month.

**Pull request closed without merging**